internal/handlers/common/validate.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"fmt"
+	"math"
+)
+
+func IsPositiveInteger(value any) error {
```

Comment:
Let's not generalize and move to common that function just yet. Let's keep it separate, but in jsonb1 package, with a name something like `validateSizeArg`. Once we have several similar fucntions, then we would generalize them.

I think that function can also use `common.NewError` instead of error strings